wiki/en/Server-Linux.md

```diff
@@ -54,6 +54,11 @@ Set your Directory (genre) server using the `--directoryserver` (`-e`) option as
 |**Genre Classical/Folk** |`classical.jamulus.io:22524`|
 |**Genre Choral/Barbershop** |`choral.jamulus.io:22724`|
 
```

Comment:
Add another example on how to register with a Custom Directory - showing how it's pretty much the same.

Reply:
I think I'm beginning to become confused about what you mean by "Custom Directory" vs "Directory". Can you explain?

Reply:
"Directory" -- A service from which a a Client can request a server list or with which a Server can register to be listed.

"Public Directory" -- Any Genre 1, Any Genre 2, Any Genre 3, Genre Rock, Genre Jazz, Genre Classical/Folk, Genre Choral/Barbershop -- the Directories built into Jamulus

"Custom Directory" -- another address entered into "Custom Directories:" in Advanced Settings on the Client or "Custom Directory server address" on the Server Opions tab.  For the Client, the user can then select that Custom Directory as a Directory in the Connect window.  For the Server, they can then choose to register with that Directory by selecting Custom.  The main difference is the Client supports multiple Custom Directories and the Server only supports a single one (as it's unlikely to change where it registers).

To run as a Directory, Jamulus is "tricked" into registering with itself using the either string "localhost" or the string "127.0.0.1" (literals, no alternatives).  It then starts allowing other Servers to register and supplying a server list to those Clients requesting one.